src/javaRosa/itextWidget.js

```diff
@@ -305,6 +305,7 @@ define([
                 if (e.form === widget.form && e.itextType === widget.itextType) {
                     if (widget.getItextValue() === e.prevValue) {
                         // Make sure all the defaults keep in sync.
+                        // why doesn't setValue set the itext value?
```

Comment:
This seems weird, but it's also been here for a while, so didn't want to rock the boat